src/plugins/GUITestBase/src/base_dialogs/GTFileDialog.h

```diff
@@ -60,11 +60,13 @@ class GTFileDialogUtils : public Filler {
     void selectFile();
     void clickButton(Button);
     void setViewMode(ViewMode);
+    void applyFilter();
 
     QWidget *fileDialog;
     QString path, fileName;
     Button button;
     GTGlobals::UseMethod method;
+    QString filter;
```

Comment:
Please document all new public fields: what is the format of this field?

Reply:
OK